src/lib/isMobilePhone.js

```diff
@@ -12,6 +12,7 @@ const phones = {
   'ar-SA': /^(!?(\+?966)|0)?5\d{8}$/,
   'ar-SY': /^(!?(\+?963)|0)?9\d{8}$/,
   'ar-TN': /^(\+?216)?[2459]\d{7}$/,
+  'bs-BA': /^((((\+|0{2})3876)|06)[0-6])((?<=4)\d{7}|(?<!4)\d{6})$/,
```

Comment:
Looks like this regex could be simplified? Please check how the rest were written...

Reply:
I can rewrite and simplify it, but with that, this regex will leave out some forms of mobile phone numbers which are valid in my country. Its your choice :-)

Reply:
Ok, I see. I'll take this in...